src/style.css

```diff
@@ -1,3 +1,96 @@
+:root {
+  --blue-color: rgba(0, 172, 220, 1);
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+  font-family: Roboto, serif;
+  font-weight: 400;
+}
```

Comment:
You are using the '*' selector to set styles for all elements, which can impact performance. Set styles only for elements that require them. Zeroing out your margins, paddings, or other styles with '*' is inefficient for the browser.